lisa/features/nvme.py

```diff
@@ -73,6 +74,16 @@ def get_namespaces(self) -> List[str]:
     def get_namespaces_from_cli(self) -> List[str]:
         return self._node.tools[Nvmecli].get_namespaces()
 
+    def get_os_partition_namespace(self) -> str:
+        node_disk = self._node.features[Disk]
+        os_partition_namespace = ""
+        os_boot_partition = node_disk.get_os_boot_partition()
+        # Sample os_boot_partition when disc controller type is NVMe:
+        # name: /dev/nvme0n1p15, disk: nvme, mount_point: /boot/efi, type: vfat
```

Comment:
The comments haven't specified what's the "namespace". Is it a general concept of disk controller?

Reply:
namespace is a concept of NVME. 
"In NVMe® technology, a namespace is a collection of logical block addresses (LBA) accessible to host software. A namespace ID (NSID) is an identifier used by a controller to provide access to a namespace. A namespace is not the physical isolation of blocks, rather the isolation of logical blocks addressable by the host software.

In Linux, namespaces appear each with a unique identifier in devices; /dev/nvme0n1 is looking at controller 0 and namespace 1)."

Please see: https://nvmexpress.org/resource/nvme-namespaces/

Reply:
The method name should be `get_os_nvme_namespace`.